### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matrix880**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,13 @@
...
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        int result = ((result1 == bet1 && result2 == bet2) && (result1 > result2 && bet1 > bet2))
+                || ((result1 == bet1 && result2 == bet2) && (result1 < result2 && bet1 < bet2))
+                || ((result1 == bet1 && result2 == bet2) && (result1 == result2 && bet1 == bet2))
+                ? 2 :
+                ((result2 > result1 && bet2 > bet1) || (result2 < result1 && bet2 < bet1)
+                        || (result1 == result2 && bet1 == bet2)) ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: don't create redundant variable. use `return condition;`

Comment (Tarabarnik) [reply src/main/java/core/basesyntax/FootballMatch.java]: complicated. try to simplify

Comment (matrix880) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        return ((result1 == bet1 && result2 == bet2)) ? 2 :
+                ((result2 > result1 && bet2 > bet1) || (result2 < result1 && bet2 < bet1)
+                        || (result1 == result2 && bet1 == bet2)) ? 1 : 0;
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1 && result2 == bet2) ? 2 :
                (result2 > result1 && bet2 > bet1) || (result2 < result1 && bet2 < bet1)
                        || (result1 == result2 && bet1 == bet2) ? 1 : 0;

```
what are this double brackets for?

Comment (matrix880) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**